platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilter.java

```diff
@@ -112,6 +102,168 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     @Override
     public void init(FilterConfig filterConfig) throws ServletException {
         this.filterConfig = filterConfig;
+
+        // now that the DelegateServletFilter has been initialized with a filterConfig, initialize all SecurityFilters using that filterConfig
+        for (ServiceReference<SecurityFilter> securityFilterServiceReference : sortedSecurityFilterServiceReferences) {
+            initFilter(securityFilterServiceReference, filterConfig);
+        }
+
+        filtersHaveBeenInitialized = true;
+    }
+
+    public void addSecurityFilter(
```

Comment:
❗️ Considering @stustison's comment about keeping around stale references and looking a little more at the implementation here...this method should definitely go away. At _most_ it should remain as a package-local method for testing purposes, but I believe even then you'd be side-stepping the true functionality that should be present.

- The `init` method should revert to what it was previously, just setting the `filterConfig` field.
- This method should be removed and you should mock/spy in your unit tests
- `doFilter` should retrieve the `SecurityFilter`s on-demand and put them into a stack-local sorted set for processing.